app-shells/fzf/Manifest

```diff
@@ -2,3 +2,5 @@ DIST fzf-0.56.3-deps.tar.xz 5884504 BLAKE2B b5ffa8dd62e81d4db0266b1c66ebcb3938dc
 DIST fzf-0.56.3.tar.gz 300405 BLAKE2B d4139fd5fc95d710f18f5415f98e512c58812d5e5849481b47898b618dd33f33cfa532a9b767a787f565337f7ca8ea383f6bf10100ef90a78bbba9ad77e68fa3 SHA512 7144036bf3d027272bb1f961bb6a2721a7a85b16955b45f67798e16ac9f7891a04ed3b343b71c3426358e9e92abe1faba10b679caf8f99ca1905c6b312e6c9b7
 DIST fzf-0.57.0-deps.tar.xz 5890984 BLAKE2B 7d1bfbcbc01f99e77d601cec1080a45d8565320f4998f1ff4691ae3afbdbd7733c5861282e6d2167bb68d02ff06331913dad53ecdce6cd2ca6ced9ba11518279 SHA512 2c46a268cea450ff224e8251354443b1f7aa7f5a939e976c2a92fa3218444504205de3a72064820df2ee25b0d4de30e7f8877d4fa004e850d82eb13f2453e9ff
 DIST fzf-0.57.0.tar.gz 303081 BLAKE2B ab478c15e9229105b73275155a54bcf3dfcfcc9704414fd84511932572156c8695982c9aee2b5f609a9ec4b82ff895a96c11bd30c5d1f1345273a7272a2d71bc SHA512 1e5a76e647bb1ab784582b1e8069df908d844e564ab1c2c7cf9d389326a97ffc3c8c102050c40656ef60039f70284ceef78b3c739d7c32ff84b7493aac71f7d2
+DIST fzf-0.59.0-deps.tar.xz 17811164 BLAKE2B fc356b71a1e80de3bcf3248b612502f096f397420cd0e6525b59cf4e0e817488a9472f39617208362f91a81b84351cbf3fa2629350b1c58a46db55e0e15e446e SHA512 2e57bef580e42c786e1f3cd89de41643207479b6a0a5add49cf3e3468c1c8a5c20f0b01118ca4f54bb4a1e4c3cb7e82b1fe83a8a45649dc663c7bc123d033951
```

Comment:
The output from an invocation of `dev-go/go-dep-tarball` is only 5897312 bytes here. Not sure if you used that.